queue_processors/autoreduction_processor/autoreduction_processor.py

```diff
@@ -136,24 +136,28 @@ class Consumer:
     """ Class used to setup the queue listener. """
     def __init__(self):
         """ Initialise consumer. """
+        # Note: The consumer name of autoreduction_processor.py is "queueProcessor"
```

Comment:
Can you change these to be the correct way round and consistent with package names (e.g. `queue_processor` and `autoreduction_processor`